emacs/README.md

```diff
@@ -0,0 +1,69 @@
+# zeek-language-server
+
+This directory contains the configuration files needed to tie into `zeek-language-server` from emacs. These instructions use `lsp-mode`. There's the possibility it would also work with `elgot` (an `lsp-mode` replacement), but that exercise is left to the reader. `lsp-mode` comes with an astonishing amount of options for configuring the look and operation of the LSP features in emacs. This guide only presents the most basic configuration.
+
+This guide assumes that you have installed `zeek-language-server` and that it is in your `PATH`.
+
+## Prerequisites
+
+There are some emacs packages that are required for this to work. You can install all of these from your favorite emacs package repository using `M-x package-install`. The required packages are:
+
+```
+use-package
+lsp-mode
+lsp-ui
+yasnippet

```

Comment:
I am not sure how familiar emacs users are with packages, but when I dropped below basic config into my system it failed (packages were not already present, and not installed automatically). When I installed them with `M-x package-install` things progressed.

Reply:
Reworded the section a little bit to mention that they should all be installed before adding the snippet.